Jenkinsfile

```diff
@@ -702,6 +702,7 @@ def withBeatsEnv(Map args = [:], Closure body) {
           error("Error '${err.toString()}'")
         } finally {
           if (archive) {
+            archiveArtifacts(allowEmptyArchive: true, artifacts: 'build/system-tests/docker-logs/TEST-docker-compose-*.log')
```

Comment:
This is my proposal, to store the artifacts `'build/system-tests/docker-logs/TEST-docker-compose-*.log'` stored in each beat.

Reply:
The CI pipeline, execute one stage per beat. Therefore I assumed `build/system-tests/docker-logs/` is a relative path for each beat that executed the system tests.

*For instance*
Given `metricbeat` 
   And guessing it can execute the system-tests
   And the workspace location is `/src/beats`
Then the full path location for those files would be `/src/beats/metricbeat/build/system-tests/docker-logs`

Reply:
Yes the docker-logs paths are as you assumed. 

If this will always save and present the log files in the list of build artifacts for each run, then that is much more convenient than the current behaviour of only uploading them to GCS on failures. 

Thanks!

Reply:
Thanks Craig, let me put some fix, since I missed the beat folder

![image](https://user-images.githubusercontent.com/2871786/147054587-abc256ee-7a97-48e3-b0c0-05215135abdf.png)

```suggestion
            archiveArtifacts(allowEmptyArchive: true, artifacts: "${directory}/build/system-tests/docker-logs/TEST-docker-compose-*.log")
```